src/components/LangSelect.js

```diff
@@ -1,13 +1,16 @@
-import { useEffect } from 'react';
 import { useLocation, useHistory } from 'react-router-dom';
-import { Button, NavList, GridContainer } from '@trussworks/react-uswds';
+import { useState } from 'react';
+import { Button, NavDropDownButton, Menu } from '@trussworks/react-uswds';
 import '@trussworks/react-uswds/lib/index.css';
 import { useTranslation } from "react-i18next";
 
 import styles from './styles.scss';
 import { LANGS } from '../lib/constants';
 
-function LangSelect({ changeLang }) {
+function LangSelect() {
+  const [isOpen, setIsOpen] = useState([false, false]);
+  const { t } = useTranslation();
+
   let history = useHistory();
   let location = useLocation();
```

Comment:
Maybe take these let statements out of the function so that these variables are only defined once rather than every time the function is called